src/main/java/com/android/volley/toolbox/DiskBasedCache.java

```diff
@@ -400,7 +400,7 @@ private CacheHeader(
                 long softTtl,
                 List<Header> allResponseHeaders) {
             this.key = key;
-            this.etag = ("".equals(etag)) ? null : etag;
+            this.etag = "".equals(etag) ? null : etag;
```

Comment:
This ended up being necessary to build and pass tests locally

Reply:
Sorry about that - forked off https://github.com/google/volley/pull/229 to keep the commits separate. Evidently our dependency on ErrorProne is not pinned to a specific version. I tried to pin it but it'd require migrating to a newer version of the plugin which appears to be more involved.